core/common/src/test/java/alluxio/wire/WorkerInfoTest.java

```diff
@@ -37,6 +37,20 @@ public void thrift() {
     checkEquality(workerInfo, other);
   }
 
+  @Test
+  public void lastContactSecComparator() {
+    int[] lastContactSecList = {55, 10, 2, 40, 100, 6};
+    int[] comparisonResults = {45, 8, -38, -60, 94};
```

Comment:
the javadoc for `compare` says
```java
     * @return a negative integer, zero, or a positive integer as the
     *         first argument is less than, equal to, or greater than the
     *         second.
```
It's an implementation detail what the actual value is. In testing the contract, we should only check the sign. 

Optionally, it might be cleaner to define a helper method that takes two lastContactSec values and compares workers with them, then have 1 line per test case like
```java
Assert.assertTrue(compareWorkersWithTimes(0, 1) < 0);
Assert.assertTrue(compareWorkersWithTimes(1, 0) > 0);
Assert.assertTrue(compareWorkersWithTimes(1, 1) = 0);
Assert.assertTrue(compareWorkersWithTimes(-1, 1) < 0);
Assert.assertTrue(compareWorkersWithTimes(1, -1) > 0);
```
This makes it easy to see which cases are being tested, and all logic is trivial (no indexing, no loops)